backend/service/topology/querybuilder.go

```diff
@@ -0,0 +1,115 @@
+package topology
+
+import (
+	"fmt"
+	"strings"
+
+	sq "github.com/Masterminds/squirrel"
+
+	topologyv1 "github.com/lyft/clutch/backend/api/topology/v1"
+)
+
+const (
+	column   = "column."
+	metadata = "metadata."
+)
+
+func paginatedQueryBuilder(
+	filter *topologyv1.SearchTopologyRequest_Filter,
+	sort *topologyv1.SearchTopologyRequest_Sort,
+	pageToken,
+	limit int,
+) (sq.SelectBuilder, error) {
+	queryLimit := 100
```

Comment:
Define this as a global const value?

Reply:
@rithujohn191, why do you think this should be global? I might be missing it but I don't think queryLimit is used outside this main func.

Reply:
I suggested it so it could be used in the test file as well and its usually considered best practice to declare default values as consts so they can be reused